src/mmw/js/src/modeling/gwlfe/runoff/templates/table.html

```diff
@@ -19,7 +19,7 @@
                     {% if loop.index0 == 0 %}
                         <td class="text-left" data-month="{{ value }}">{{ monthNames[value] }}</td>
                     {% else %}
-                        <td class="strong text-right">{{ value|round(2)|toLocaleString(2) }}</td>
```

Comment:
Switched from using a `text-right` class to `data-align="right"` on the `th` so that the right alignment carries through to the footer (which is in another table).